modules/microshift-adding-olm-to-blueprint.adoc

```diff
@@ -0,0 +1,32 @@
+// Module included in the following assemblies:
+//
+// * microshift/microshift-update-rpms-ostree.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="microshift-adding-olm-to-blueprint_{context}"]
+== Adding the Operator Lifecycle Manager (OLM) service to a blueprint 
+
+When you install {microshift-short}, the Operator Lifecycle Manager (OLM) package is not installed by default. You can add the `microshift-olm` package in the ostree blueprint to enable OLM in {microshift-short}.
+
+. Edit your ostree blueprint by running the following example command:
++
+[source,terminal]
+----
+$ vi <minimal-microshift>.toml <1>
+----
+<1> Specify the name of the blueprint file you used when adding the MicroShift service.
+
+. Add the following example text to your ostree blueprint:
++
+[source,text]
+----
+[[packages]]
+name = "microshift-olm"
+version = "*"
+----
+. To apply the manifest from the package to an active cluster, run the following command: 
```

Comment:
Oh, this isn't that simple. When you update the osbuild blueprint you need to build new ostree commit then deploy it on the machine (and reboot)

Reply:
Gotcha, its never as simple as one command lol. How would a user build a new ostree commit?

Reply:
We already have https://70359--ocpdocs-pr.netlify.app/microshift/latest/microshift_install/microshift-embed-in-rpm-ostree and https://70359--ocpdocs-pr.netlify.app/microshift/latest/microshift_updating/microshift-update-rpms-ostree so we might link to them